### PR TITLE
New "Asking for feedback" page

### LINKED ISSUE #38: Asking users for feedback
A feedback page or form in the service. Most commonly a 3rd party product is being used at the moment.

### PR DESCRIPTION
**Pull request by ahosgood**
Closes #38

### CODE REVIEW

File: src/patterns/asking-for-feedback/index.md.njk [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+2. A content designer should lead the design of the form.
+3. Use an interaction designer if you need to, to ensure existing proven patterns are used or to create new patterns.
+
+The rest of the development team should follow their same responsibilities they would for any other part of your service. For example, the feedback form must be accessibility tested for compliance.
+
+### How to decide between third party products or building your own
+
+Off-the shelf surveys and form products, for example Microsoft forms and Google forms, are often used for ‘collecting information’.
+
+The laws that apply to pages we build ourselves equally apply to third-party ones. For example, we are responsible for following data protection laws and the Equality Act 2010. .
```
Comment (AndyPorter) [inline src/patterns/asking-for-feedback/index.md.njk:66]: i think we can make the slime clearer - and remove the additional full stop.

"The laws that apply to pages we build ourselves equally apply to third-party ones. For example, we are responsible for following data protection laws and the Equality Act 2010. ."


I suggest saying...
"The laws that apply to pages we build ourselves apply equally to those we choose to include from third parties. We, as the operator of the service, are responsible for those third party pages (or sub services) meeting the laws.  For example data protection laws and the Equality Act 2010."

"If outsourcing data collection and processing, you will probably need a lawyer to help fully understand the data protection laws in the UK and assess the potential partners’ privacy and security procedures."

File: src/patterns/asking-for-feedback/index.md.njk [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+This applies to internal services as well as public and professional facing services.
+
+This applies to people using both the digital and non-digital parts of your service.
+
+How to:
+
+- [Make your service Inclusive (non-discriminating)](https://www.gov.uk/service-manual/design/making-your-service-more-inclusive)
+- [Provide accessible services and assisted digital support](https://www.gov.uk/service-manual/helping-people-to-use-your-service)
+
+Does a service have to do it? (Welsh – think this lives n the Welsh guidance)
```
Comment (AndyPorter) [inline src/patterns/asking-for-feedback/index.md.njk:142]: remove the following content from this page

Does a service have to do it? (Welsh – think this lives n the Welsh guidance)
Dunno - to write. Ask a lawyer, ask them early on - e.g., early in Alpha."

**Review by AndyPorter — CHANGES_REQUESTED** [review]
consider my comment on line 66 about making the point on our responsibility for third parties clearer.  But at least remove the additional full stop on the current line 66

remove lines 142 to 145 - comment on line 142 indicates what content to remove

**Review by accessibilitydesign — APPROVED** [review]

File: src/patterns/asking-for-feedback/index.md.njk [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+2. A content designer should lead the design of the form.
+3. Use an interaction designer if you need to, to ensure existing proven patterns are used or to create new patterns.
+
+The rest of the development team should follow their same responsibilities they would for any other part of your service. For example, the feedback form must be accessibility tested for compliance.
+
+### How to decide between third party products or building your own
+
+Off-the-shelf surveys and form products, for example Microsoft forms and Google forms, are often used for collecting information.
+
+The laws that apply to feedback pages we code ourselves apply equally to those we choose to include from third parties. As the operator of the service, we are responsible for those third parties compliance with the law. For example the data protection laws and the Equality Act 2010.
```
Comment (ahosgood) [inline src/patterns/asking-for-feedback/index.md.njk:67]: Not sure I understand what this paragraph is trying to say. If we use a third party tool (e.g. Google Analytics) then it isn't our responsibility to ensure their compliance.

File: src/patterns/asking-for-feedback/index.md.njk [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+The rest of the team should consider this page as they would any other in the services, for example:
+
+- lawyers confirming compliance with all laws
+- testers ensuring its technical accessibility
+- software developers should ensure, for example it’s secure, is available, what happens if the service (or part of) is not available.
+- user researchers and designers to ensure its usable and accessible (non-technical compliance
+
+The feedback form must be accessibility tested as part of the accessibility statement.
+
+## Accessibility and inclusivity
```
Comment (ahosgood) [inline src/patterns/asking-for-feedback/index.md.njk:129]: Is this somewhat a duplication of what is in the Accessibility section of the Design System?

**Pull request closed without merging**